gcinvoice.py

```diff
@@ -326,7 +326,7 @@ def parse(self, gcfile=None):
                 entrydict['guid'] = entry.findtext(ns('entry:guid'))
                 entrydict['date'] = _readdate(entry.findtext(
                     ns('entry:date/ts:date')))
-                entrydict['entered'] = _readdatetime(entry.findtext(
+                entrydict['entered'] = _readdate(entry.findtext(
```

Comment:
Why do you want the change the entered into a date? This changes the sorting of the gnucash test entries.